src/util-ebpf.c

```diff
@@ -258,11 +412,31 @@ int EBPFLoadFile(const char *iface, const char *path, const char * section,
         SCLogDebug("Got a map '%s' with fd '%d'", bpf_map__name(map), bpf_map__fd(map));
         bpf_map_data->array[bpf_map_data->last].fd = bpf_map__fd(map);
         bpf_map_data->array[bpf_map_data->last].name = SCStrdup(bpf_map__name(map));
+        snprintf(bpf_map_data->array[bpf_map_data->last].iface, IFNAMSIZ,
+                 "%s", iface);
         if (!bpf_map_data->array[bpf_map_data->last].name) {
             SCLogError(SC_ERR_MEM_ALLOC, "Unable to duplicate map name");
             BpfMapsInfoFree(bpf_map_data);
             return -1;
         }
+        bpf_map_data->array[bpf_map_data->last].unlink = 0;
+        if (config->flags & EBPF_PINNED_MAPS) {
+            SCLogNotice("Pinning: %d to %s", bpf_map_data->array[bpf_map_data->last].fd,
```

Comment:
Config?

Reply:
Well potential error can be system error like bpf pseudo filesystem not begin mounted. Should I use Config for that ?